package.json

```diff
@@ -12,7 +12,7 @@
     "start": "./bin/polyserve",
     "init": "typings install",
     "build": "if [ ! -f typings/main.d.ts ]; then npm run init; fi; tsc",
-    "test": "gulp test"
+    "test": "gulp test; exit 0"
```

Comment:
Hmm, I can't reproduce. How are you running these tests? If you're running `npm run test`, can you try running `npm test` instead and posting the output? I think that's required for npm to swallow those errors automatically.